.github/workflows/integration-test-v1.yml

```diff
@@ -50,23 +45,17 @@ jobs:
 
       - name: Determine env variables
         run: |
-          if [ "${{ github.event.inputs.environment }}" == 'staging' ]; then
-             echo "USERNAME=${{ secrets.FIREBOLT_USERNAME_STAGING }}" >> "$GITHUB_ENV"
-             echo "PASSWORD=${{ secrets.FIREBOLT_PASSWORD_STAGING }}" >> "$GITHUB_ENV"
-             echo "EXCLUDE_ENV=dev" >> "$GITHUB_ENV"
-          else
-             echo "USERNAME=${{ secrets.FIREBOLT_USERNAME_DEV }}" >> "$GITHUB_ENV"
-             echo "PASSWORD=${{ secrets.FIREBOLT_PASSWORD_DEV }}" >> "$GITHUB_ENV"
-             echo "EXCLUDE_ENV=staging" >> "$GITHUB_ENV"
-          fi
+           echo "USERNAME=${{ secrets.FIREBOLT_USERNAME_STAGING }}" >> "$GITHUB_ENV"
+           echo "PASSWORD=${{ secrets.FIREBOLT_PASSWORD_STAGING }}" >> "$GITHUB_ENV"
```

Comment:
We need to remove this step altogether, as it breaks when special characters are encountered. You can pass the secrets directly e.g. https://github.com/firebolt-db/dbt-firebolt/pull/120/files#diff-29ce8407d15551c9c62184e207b6ceadebb261f7ffbfb75d7d67844ba33709abR33

Reply:
Hopefully fixed. Please review again.